mongo/filterstore.go

```diff
@@ -21,9 +21,7 @@ var (
 	errOptionNotFound            = errors.New("Option not found")
 	errFilterOrDimensionNotFound = errors.New("Bad request - filter or dimension not found")
 	errFilterOutputNotFound      = errors.New("Filter output not found")
-
-	errFilterBadRequest    = errors.New("Bad request - filter blueprint not found")
-	errDimensionBadRequest = errors.New("Bad request - filter dimension not found")
+	errFilterBadRequest          = errors.New("Bad request - filter blueprint not found")
```

Comment:
I feel like this should not exist, if filter blueprint is not found then it is a 404?